src/auto_pickup.cpp

```diff
@@ -112,87 +115,87 @@ void show_auto_pickup()
     ctxt.register_action("MOVE_RULE_UP");
     ctxt.register_action("MOVE_RULE_DOWN");
     ctxt.register_action("TEST_RULE");
-    ctxt.register_action("SWITCH_AUTO_PICKUP_OPTION");
+    ctxt.register_action("SWITCH_OPTION");
```

Comment:
The default key binding for this action (in data/raw/keybindings.json) hasn't been changed, it still binds the key to the action "SWITCH_AUTO_PICKUP_OPTION" (which is not used anywhere). This change also makes the user-specific key binding invalid (users have to re-enable this binding). I suggest not changing the action strings at all. It may look ugly, but it's better for compatibility.